RadixWallet/Features/HomeFeature/Coordinator/Home.swift

```diff
@@ -92,6 +98,18 @@ public struct Home: Sendable, FeatureReducer {
 
 	public func reduce(into state: inout State, viewAction: ViewAction) -> Effect<Action> {
 		switch viewAction {
+		case .onFirstAppear:
+			if iOSSecurityClient.isJailbroken() {
+				state.destination = .acknowledgeJailbreakAlert(.init(
+					title: .init("Possible jailbreak detected"), // FIXME: Strings - get real string
+					message: .init("It appears that your device might be jailbroken. To ensure the security of your Accounts and assets, using the Radix Wallet on jailbroken devices is not recommended. Please confirm if you wish to continue anyway at your own risk."), // FIXME: Strings - splash_rootDetection_title - should be messageIOS
+					buttons: [
+						.cancel(.init("I Understand the Risk")),
```

Comment:
FIXME: Strings on the button title as well?

Reply:
yes